src/Site.js

```diff
@@ -899,9 +900,21 @@ class Site {
       return Promise.resolve();
     }
     return new Promise((resolve, reject) => {
-      fs.copyAsync(siteLayoutPath, layoutsDestPath)
-        .then(resolve)
-        .catch(reject);
+      glob(`${siteLayoutPath}/**/*`, (err, res) => {
```

Comment:
We actually already have `walk-sync` which we use to walk through the files and directories. We can use that instead of using a brand new package.